client/src/components/DetailItem.jsx

```diff
@@ -1,25 +1,26 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import styles from '../scss/base.scss';
 
 const DetailItem = ({ title, icon, details }) => {
   if (details) {
     let link;
     if (title === 'Address') {
```

Comment:
consider using a switch statement instead. i think it would look more graceful :)

Reply:
I don't like to use switch statements when things aren't explicitly returning - https://hackernoon.com/rethinking-javascript-eliminate-the-switch-statement-for-better-code-5c81c044716d